test/image/mocks/canada_geo_projections.json

```diff
@@ -0,0 +1,5753 @@
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo46": {
+      "domain": {
+        "row": 5,
+        "column": 5
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "kavrayskiy7"
+      },
+      "lonaxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo47": {
+      "domain": {
+        "row": 6,
+        "column": 5
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "lagrange"
+      },
+      "lonaxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo48": {
+      "domain": {
+        "row": 7,
+        "column": 5
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "larrivee"
+      },
+      "lonaxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo49": {
+      "domain": {
+        "row": 0,
+        "column": 6
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "laskowski"
+      },
+      "lonaxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo50": {
+      "domain": {
+        "row": 1,
+        "column": 6
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "loximuthal"
+      },
+      "lonaxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "lataxis": {
+        "showgrid": true,
+        "gridwidth": 1,
+        "gridcolor": "lightgray"
+      },
+      "showcoastlines": false,
+      "showocean": true
+    },
+    "geo51": {
+      "domain": {
+        "row": 2,
+        "column": 6
+      },
+      "fitbounds": "locations",
+      "projection": {
+        "rotation": {"lat": 45},
+        "type": "mercator"
```

Comment:
strange to think of this as a mercator projection when it has a rotation... can we omit `projection.rotation` from this mock, and let the projection do its own thing? Hopefully that will automatically rotate those projections that use rotation to fit their contents, but will use `center` for mercator and others like it. That would allow this mocks to function as more than just "these projections all work zoomed in", but also "here's the kind of map you can expect from each projection"

Reply:
Hmm... `geo_fitbounds-locations` basically tests just that for Australia.
OK please see https://github.com/plotly/plotly.js/pull/5801/commits/77f03674220ed2870afde9637d92696f60e3755b and then https://github.com/plotly/plotly.js/pull/5801/commits/8e352e2c9cc8e9971998e2819b73b57ff7e9989d.
Keeping rotation for those three projections can simply help illustrate that the projection could actually be used with `Canada`.

Reply:
> Keeping rotation for those three projections can simply help illustrate that the projection could actually be used with Canada.

I'd argue that this shows these projections may not be optimizing the correct variables when doing `fitbounds` - but if there's no obvious fix for that we can leave it. Thanks for dropping all the other rotations.